qbraid/runtime/rigetti/provider.py

```diff
@@ -0,0 +1,92 @@
+import os
+from typing import List
```

Comment:
Don't use `typing.List`, just use generic `list` for list type annotations. See [PEP 585](https://peps.python.org/pep-0585/)